src/subcommands/traits.rs

```diff
@@ -141,6 +143,13 @@ pub trait FinalfrontierApp {
                     .help("Number of threads (default: min(logical_cpus / 2, 20))")
                     .takes_value(true),
             )
+            .arg(
```

Comment:
It's best to put this in a group of mutually exclusive options with `MINCOUNT`. Then `clap` ensures that the user does not use both options at the same time.